qa/L0_mlflow/test.sh

```diff
@@ -63,8 +63,12 @@ EOF
 
 rm -rf ./models
 mkdir -p ./models
+# Put some models in model repository to make sure MLFlow plugin would ignore
+# model that is not registered via MLFlow
+cp -r ./mlflow-triton-plugin/examples/onnx_float32_int32_int32 ./models/existing_model
+
 SERVER=/opt/tritonserver/bin/tritonserver
-SERVER_ARGS="--model-repository=./models --strict-model-config=false --model-control-mode=explicit"
+SERVER_ARGS="--model-repository=./models --strict-model-config=false --model-control-mode=explicit --load-model=*"
```

Comment:
Just curious why `--load-model=*` is needed here.

```python
resp = self.triton_client.get_model_repository_index()
```

The above client call with default parameters could list all models.